.changeset/tasty-turkeys-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+Preloaded inter font on Storybook/Chromatic
```

Comment:
I think we can remove this changeset since these changes don't impact consumers of `@shopify/polaris`